### PR TITLE
Update build-windows-toolchain.bat

### PR DESCRIPTION
**Pull request by compnerd**
Attempt to repair the CI hosts where the extraction of WiX would fail due to the file existing.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

File: utils/build-windows-toolchain.bat [hunk 1]
```diff
@@ -855,8 +855,9 @@ endlocal
 :FetchWiX
 setlocal enableextensions enabledelayedexpansion
 
+if exist wix-4.0.1 goto :eof
```
Comment (etcwilde) [inline utils/build-windows-toolchain.bat:858]: what is `:eof`? Don't we want to just finish this function and keep going with the toolchain build?

Comment (etcwilde) [reply utils/build-windows-toolchain.bat:858]: Oh, eof, "end-of-function", not "end-of-file".
Have to love common terms being abbreviated to the same thing while meaning different things.

**Review by etcwilde — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please build toolchain windows platform

**Pull request merged**